app/views/order_mailer/order_email.html.erb

```diff
@@ -0,0 +1,13 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <h1>Hi <%= @order.phone %></h1>
+    <p>
+      Thank you for your order! Just wait for a call from our dispatcher!
```

Comment:
Maybe it's a good idea to add order id somewhere in the letter's subject or body.